cmd/tracee-ebpf/internal/flags/flags-containers.go

```diff
@@ -0,0 +1,116 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/containers/runtime"
+)
+
+func ContainersHelp() string {
+	return `Select which container runtimes to connect to for container events enrichment.
+By default, if no flag is passed, tracee will automatically detect installed runtimes by going through known runtime socket paths.
+
+Tracee will look for the following paths:
+1. Docker:     /var/run/docker.sock
+2. Containerd: /var/run/containerd/containerd.sock
+3. CRI-O:      /var/run/crio/crio.sock
+
```

Comment:
Don't we want to add podman as well? What is missing in order to add such a support?

Reply:
Yes, but I've decided to leave it to a later PR, since it requires implementing our own client since I couldn't find a good client library (their own library is pretty bad IMO, it's version tied and has a ton of unrelated dependencies).